src/osd/PrimaryLogPG.cc

```diff
@@ -6795,6 +6795,10 @@ int PrimaryLogPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
 	  map<string,bufferlist> rmattrs;
 	  result = getattrs_maybe_cache(ctx->obc, &rmattrs);
 	  if (result < 0) {
+<<<<<<< HEAD
```

Comment:
@NancySu05 mind resolving the merge conflicting?

Reply:
sorry, I'v already resolved conflicting, please check.